service/src/main/java/bio/terra/workspace/service/features/FeatureService.java

```diff
@@ -0,0 +1,26 @@
+package bio.terra.workspace.service.features;
+
+import bio.terra.common.flagsmith.FlagsmithService;
+import bio.terra.workspace.common.exception.FeatureNotSupportedException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class FeatureService {
+  private final FlagsmithService flagsmithService;
+
+  @Autowired
+  FeatureService(FlagsmithService flagsmithService) {
+    this.flagsmithService = flagsmithService;
+  }
+
+  public boolean awsEnabled() {
```

Comment:
@ddietterich we can just inject featureconfiguration in here for features that broad would care about. but i don't think they care about aws atm so I don't want to do that. 

if it's a broad only flag, they can just keep using featureconfigurations. as long as application.yaml keep it as false by default, we don't need to worry about it.